pkg/config/public_cloud_specs_test.go

```diff
@@ -194,6 +195,65 @@ func TestGetFeature(t *testing.T) {
 	}
 }
 
+func TestGetFeatureFractional(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+	config := &env.Config{PublicCloudSpecsPath: testPublicCloudSpecsPathFractional}
+	specs, err := LoadPublicCloudSpecs(config)
+	g.Expect(err).Should(gomega.BeNil())
+
+	testCases := []struct {
+		cloudProvider   string
+		vmType          string
+		expectedFeature Feature
+		wantNil         bool
```

Comment:
I can see that the `wantNil` is not used in any of the test cases, so we can remove it here, right?

Reply:
jump